lib/howl/bindings.moon

```diff
@@ -211,12 +219,17 @@ export keystrokes_for = (handler, source = nil) ->
 
   keystrokes
 
-export action_for = (translation, source='editor') ->
+export action_for = (tr, source='editor') ->
+  os = sys.info.os
+  empty = {}
```

Comment:
Well, it adds some descriptive value I find, but also I cringed a little bit at creating so many empty tables right after one another, when they needn't be different.